tox.ini

```diff
@@ -9,7 +9,7 @@ basepython = python2.7
 deps =
     -r{toxinidir}/requirements.txt
     pytest-cov==2.6.1
-    pytest==4.1.0
+    pytest==4.3.0
```

Comment:
The latest pytest version that works with py2 is 4.6.5, can you upgrade to that one?

Reply:
Done.